common-lib/src/main/java/com/google/gct/idea/feedback/FeedbackUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.google.gct.idea.feedback;
+
+import com.intellij.ExtensionPoints;
+import com.intellij.ide.plugins.PluginManager;
+import com.intellij.openapi.extensions.Extensions;
+import com.intellij.openapi.extensions.PluginId;
+
+/**
+ * Handy plugin related methods for Google Feedback integration.
+ */
+public final class FeedbackUtil {
+
+  private FeedbackUtil() {}
+
+  public static void enableGoogleFeedbackErrorReporting(String pluginId) {
```

Comment:
Yeah sadly it doesn't work.  I'll come back round to this in a few days as soon as I've synced core and login into AOSP.  I want to get my head around testing and have a clear strategy for managing collaborators and making our code more testable and tested, but I don't want to block on that to get this in, since it's needed for the sync to AOSP, and I've manually verified that it works.